frb_codegen/src/library/codegen/generator/api_dart/spec_generator/class/field.rs

```diff
@@ -51,11 +51,19 @@ fn default_value_maybe_to_dart_style(value: &str, enable: bool) -> Cow<str> {
 }
 
 fn default_value_to_dart_style(value: &str) -> String {
-    let mut split = value.split('.');
-    let enum_name = split.next().unwrap();
-
-    let variant_name = split.next().unwrap().to_string();
-    let variant_name = make_string_keyword_safe(variant_name.to_case(Case::Camel));
-
-    format!("{enum_name}.{variant_name}")
+    match value.split_once('.') {
+        // If the user is explicitly calling an enum variant's constructor
```

Comment:
nit: would be great to add a quick unit test for this function (just inside this file and a few lines)